src/lib.rs

```diff
@@ -65,7 +76,8 @@
 extern crate difference;
 #[macro_use] extern crate error_chain;
 
-use std::process::Command;
+use std::process::{Command, Output};
+use std::fmt;
```

Comment:
ok, that sounds good. But when `pub` and `mod` come into play I find it less clear - I usually end up with
```
use std
use external
pub use mystuff
use myotherstuff

pub mod mymod
mod myothermod
```

But I've also seen `mod use` mixed like here.